### ISSUE TITLE
Repetition of "Issues" in Opinions and Skeletons

### ISSUE THREAD

**Issue by passamo9** [passamo9/LaTeX_EnglandWales_Templates#33]
I wanted to raise something that I had mentioned yesterday.

In all Skeletons, and most Opinions, lawyers are asked to identify at the outset "Issues", then answer those issues with the Test (as applicable legal provisions are known) and evidence in support.

### Opinions

[Yesterday](https://github.com/passamo9/LaTeX_EnglandWales_Templates/pull/10#issuecomment-778783758) I linked to the Three Knights opinion, which had in Para 1 three issues: (i), (ii), (iii).

**On page 7, issue (i) becomes it's own heading, but crucially it's treated in a very understated way.**

><img width="570" alt="image" src="https://user-images.githubusercontent.com/69139574/108000690-d4e8f780-6fea-11eb-9eea-3a0af88087c7.png">

**Same thing on page 15:**
><img width="570" alt="image" src="https://user-images.githubusercontent.com/69139574/108000730-efbb6c00-6fea-11eb-809d-a1fe32e63302.png">

**And page 23:**
><img width="570" alt="image" src="https://user-images.githubusercontent.com/69139574/108000755-fc3fc480-6fea-11eb-8e58-03dd04d4ce8c.png">

### Skeletons

Here's a skeleton that I wrote for my bar finals.

[ADVexam_FINAL2.pdf](https://github.com/passamo9/LaTeX_EnglandWales_Templates/files/5984564/ADVexam_FINAL2.pdf)

1. Paragraphs 1 and 2 are self-explanatory

2. Documents is a kettle of fish—let's not do this here.

3. Para 4 is the "Test", or what legal provisions will apply. If you skip ahead to paras 6 - 11, you will see that the provisions in paras 4.1 - 4.4. are the new paras 6 - 11.

4. This is the crucial part: you will see that the issues in 5.1 (a) - (d) are, like in the Three Knights opinion, repeated where they are applicable, and in fact, _even though they are italicised and do not follow the paragraph numbering, they are the actual (underlying) structure of everything_ that follows after the paragraph in which they are announced.

Which brings me to my point.

Is it conceivable to announce issues e.g. "issue1", so that then whenever they need to be repeated, the author could write "issue 1" and the text of the issue would not have to be rewritten?

Please let me know if this is too ambitious, and needs to be scrapped.

**Comment by yjyao** [comment]
yes it's feasible. we need exactly this kind of feature requests because we will never know what you need since we won't be writing the docs. let's keep this issue open until someone picks it up.

**Comment by passamo9** [comment]
Glad to hear!
